systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaUserResource.java

```diff
@@ -47,6 +47,9 @@ public boolean waitForReadiness(KafkaUser resource) {
     public static MixedOperation<KafkaUser, KafkaUserList, Resource<KafkaUser>> kafkaUserClient() {
         return Crds.kafkaUserOperation(ResourceManager.kubeClient().getClient());
     }
+    public void update(KafkaUser kafkaUser) {
```

Comment:
Why does this one not `@Override` the parent?

Reply:
Sorry my mistake, fixed, thank you for noticing.